[2022-03-18 21:04:06,964] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: xcom_dag.processing_tasks.training_model_c 2022-03-17T00:00:00+00:00 [queued]>
[2022-03-18 21:04:07,027] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: xcom_dag.processing_tasks.training_model_c 2022-03-17T00:00:00+00:00 [queued]>
[2022-03-18 21:04:07,028] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 21:04:07,028] {taskinstance.py:1068} INFO - Starting attempt 3 of 3
[2022-03-18 21:04:07,030] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 21:04:07,052] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): processing_tasks.training_model_c> on 2022-03-17T00:00:00+00:00
[2022-03-18 21:04:07,059] {standard_task_runner.py:52} INFO - Started process 7124 to run task
[2022-03-18 21:04:07,066] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'xcom_dag', 'processing_tasks.training_model_c', '2022-03-17T00:00:00+00:00', '--job-id', '43', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/xcom_dag.py', '--cfg-path', '/tmp/tmpjznmfrmg', '--error-file', '/tmp/tmp0_jyaall']
[2022-03-18 21:04:07,070] {standard_task_runner.py:77} INFO - Job 43: Subtask processing_tasks.training_model_c
[2022-03-18 21:04:07,206] {logging_mixin.py:104} INFO - Running <TaskInstance: xcom_dag.processing_tasks.training_model_c 2022-03-17T00:00:00+00:00 [running]> on host pc
[2022-03-18 21:04:07,334] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=xcom_dag
AIRFLOW_CTX_TASK_ID=processing_tasks.training_model_c
AIRFLOW_CTX_EXECUTION_DATE=2022-03-17T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-17T00:00:00+00:00
[2022-03-18 21:04:07,335] {logging_mixin.py:104} INFO - model's accuracy: 6.0105519608166285
[2022-03-18 21:04:07,527] {python.py:151} INFO - Done. Returned value was: None
[2022-03-18 21:04:07,606] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=xcom_dag, task_id=processing_tasks.training_model_c, execution_date=20220317T000000, start_date=20220318T153406, end_date=20220318T153407
[2022-03-18 21:04:07,727] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-03-18 21:04:07,765] {local_task_job.py:151} INFO - Task exited with return code 0
